concepts/staking.md

```diff
@@ -6,40 +6,26 @@ description: Details of Staking with Angle Protocol
 
 ## 🔎 TL;DR
 
-A portion of Angle's governance tokens will be distributed to stable holders, SLPs and HAs through staking contracts.
+A portion of Angle's governance tokens are distributed to protocol and agToken users.
 
 ## 💡 Rationale
 
 Angle aims to be a decentralized protocol. To this extent, the protocol will only succeed if the ownership of the governance token backing the protocol is decentralized.
 
-A vast portion of the governance tokens are distributed through staking contracts by the protocol to Standard Liquidity Providers (in majority), Hedging Agents and agTokens holders.
+A vast portion of the governance tokens are distributed through staking contracts by the protocol to protocol and agToken users. This include Hedging Agents, SLPs, and agToken LPs on specific pools.
 
 While Hedging Agents automatically can accumulate governance tokens automatically as they stay in the protocol, Standard Liquidity Providers as well as stable holders have to stake their tokens (sanTokens for SLPs and agTokens for stable holders) in specific contracts to earn governance tokens. The protocol leaves them the opportunity to stake directly and seamlessly from the app after having minted.
 
-{% hint style="info" %}
-Staking contracts of the protocol may not be released immediately at launch of the protocol on mainnet.
-{% endhint %}
-
 ## 💐 Token Distribution
 
-There is one specific staking contract per sanToken and per agToken. The staking contracts for Hedging Agents are directly the contracts in which HAs' positions are stored.
-
-The fewer tokens are staked, the more interesting it is to participate in this staking contract because a similar amount of governance tokens are shared among a smaller amount of people.
+The distribution of governance tokens between the different staking contracts (called gauges) happens through weekly votes from veANGLE token holders. They assign specific weights to the different gauges, and the sum of their votes will dictate the next rewards distribution.
```

Comment:
Here we should add more references to gauges and gauges voting.
Vote is not really weekly, it's just a Snapshot that's taken weekly but you can decide not to vote for like a month